api/registrations/serializers.py

```diff
@@ -117,7 +117,7 @@ class RegistrationSerializer(NodeSerializer):
         related_view_kwargs={'node_id': '<_id>'},
         related_meta={
             'unread': 'get_unread_comments_count',
-            'total': 'get_total_comments_count',  # total count of top_level, undeleted comments on the registration (node)
+            'count': 'get_total_comments_count',
```

Comment:
Isn't this a breaking change? Is this a documented property?

Reply:
Nope, `total` was added earlier in this PR. [Previously it had just `unread`](https://github.com/CenterForOpenScience/osf.io/pull/8876/files#diff-c4e5a8ca4d9df4503422cdf682b3235bL114)

Reply:
(`related_counts` causes special handling of `related_meta['unread']` and `related_meta['count']`, so `total` was *always* included. Switching to `count` seemed cleaner than adding yet another special case)

Reply:
Ah, makes sense. Thanks for clarifying.